python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -1435,7 +1435,7 @@ def test_find_version(lmdb_version_store_v1):
     with pytest.raises(NoDataFoundException):
         lib._find_version(sym, as_of="snap_1000")
     # By timestamp
-    assert lib._find_version(sym, as_of=time_0).version == 0
+    # assert lib._find_version(sym, as_of=time_0).version == 0
```

Comment:
What are we commenting this assertion out?

Reply:
It's back